lib/data_magic.rb

```diff
@@ -113,10 +121,30 @@ def self.search(terms, options = {})
     end
 
     # assemble a simpler json document to return
+    simple_result =
     {
       "metadata" => metadata,
       "results" => 	results
     }
+
+    if options[:command] == 'stats'
```

Comment:
This should be broken out into its own method. (We should refactor the rest of `search()` anyway, it doesn't have to be done now)